front/src/common/BootstrapSNCF/InputGroupSNCF.tsx

```diff
@@ -125,7 +127,7 @@ export default function InputGroupSNCF({
         value={inputValue}
         min={min}
         max={max}
-        data-testid="input-group-first-field"
+        data-testid={inputDataTestId}
```

Comment:
Why use camelCase here where you use kebab-case above?

Reply:
Here camelCase is for the props variable